packages/paste-core/components/timeline/src/TimelineItemGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, type BoxProps } from "@twilio-paste/box";
+import { DetailText } from "@twilio-paste/detail-text";
+import * as React from "react";
+
+import { TimelineContext, TimelineGroupContext } from "./TimelineContext";
+import { TimelineItemIcon } from "./TimelineItemIcon";
+import type { TimelineItemProps } from "./types";
+
+export const TimelineItemGroup = React.forwardRef<
+  HTMLDivElement,
+  Omit<TimelineItemProps, "title"> & {
+    /**
+     * Overrides the default element name to apply unique styles with the Customization Provider.
+     *
+     * @default "TIMELINE_ITEM_GROUP"
+     * @type {BoxProps["element"]}
+     * @memberof TimelineProps
+     */
+    element?: BoxProps["element"];
+  }
+>(({ children, timestamp, icon, element = "TIMELINE_ITEM_GROUP", ...props }, ref) => {
+  const { orientation } = React.useContext(TimelineContext);
+
+  if (!orientation) {
+    throw new Error("TimelineItemGroup must be used within a Timeline component");
+  }
+
+  return (
+    <Box
+      element={element}
+      ref={ref}
+      as="li"
+      position="relative"
+      display="flex"
+      columnGap="space50"
+      rowGap="space50"
+      flexShrink={0}
+      flexDirection={orientation === "horizontal" ? "column" : "row"}
+      paddingX="space0"
+      {...props}
+    >
+      <Box
+        element={`${element}_SEPARATOR`}
+        display="flex"
+        width={orientation === "horizontal" ? "initial" : "sizeBase50"}
+        flexDirection={orientation === "horizontal" ? "row" : "column"}
+        alignItems="center"
+      >
+        <TimelineItemIcon icon={icon} />
+      </Box>
+
+      <Box element={`${element}_CONTENT`}>
```

Comment:
suggestion: let's call this and the Box on line 59 two different things (element_CONTENT)